packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -178,4 +178,8 @@ contract Bridge is EssentialContract, IBridge {
     ) public pure returns (bytes32) {
         return LibBridgeStatus.getMessageStatusSlot(msgHash);
     }
+
+    function isEtherReleased(bytes32 msgHash) public view returns (bool) {
```

Comment:
Please move this function before any pure function, otherwise running 'pnpm lint:sol' will have warnings.

Reply:
I edited it.